src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -1635,16 +1635,17 @@ void Session::send_bind_message()
 
     session_ident_type session_ident = m_ident;
     const std::string& path = get_virt_path();
-    const std::string& signed_access_token = get_signed_access_token();
```

Comment:
is get_signed_access_token() used anywhere? if not we can remove it.

Reply:
good point. it is indeed not used anymore. I'll remove it.